### DISCUSSION TITLE
How to help for newbi

### DISCUSSION THREAD

**Discussion by addboy58** [hatlabs/discussions#84] [General]
Any place you can find a how to step by step on how to load the software? I looked on YouTube and did see anything.
Thx
D

**Comment by Rjn9533** [comment]
Wouldn't it be great if someone would take the time to do a video on how to install the firmware on the Hamlet from the time you recieve it in the mail to uploading the firmware on the board, then go through the process of adding things like 1-wire temp sensors, and editing the code and the same with IC2 sensors and all the other inputs.   

I know this sounds like lots of work but for rookies like us it would be so appreciated.

**Comment by big-col** [comment]
For which device do you require help?
